python/dgl/runtime/scheduler.py

```diff
@@ -746,26 +790,36 @@ def _gen_send_reduce(
     var_send_edges : var.IDX
         The edges (ids) to perform send.
     var_reduce_nodes : var.IDX
-        The nodes to perform reduce. This should include unique(v) + 0deg nodes.
+        Unique and sorted nodes to perform reduce. This should include
+        unique(v) + 0deg nodes.
     uv_getter : callable
-        A function that returns a pair of var.IDX (u, v) for the triggered edges.
+        Function that returns a pair of var.IDX (u, v) for the triggered edges.
     adj_creator : callable
-        A function that returns the adjmat and the shuffle index.
-    inc_creator : callable
-        A function that returns the incmat and the shuffle index.
+        Function that returns the adjmat, edge order of csr matrix, and edge
+        order of transposed csr matrix
```

Comment:
The third return of adj_creator() seems to be bit width instead of transposed CSR.